sched/task/exit.c

```diff
@@ -108,3 +106,8 @@ void exit(int status)
 
   _exit(status);
 }
+
+void exit(int status)
```

Comment:
why provide two exit implementation?

Reply:
Could you explain more about error? We need fix it directly, instead work around it.

Reply:
It is this one I think:
 ```
====================================================================================
Configuration/Tool: amber/hello
------------------------------------------------------------------------------------
  Cleaning...
  Configuring...
  Building NuttX...
/usr/lib/gcc/avr/5.4.0/avr51/libgcc.a(_exit.o): In function `_exit':
(.fini9+0x0): multiple definition of `_exit'
/github/workspace/sources/nuttx/staging/libsched.a(exit.o):exit.c:(.text+0x0): first defined here
make[1]: *** [Makefile:94: nuttx] Error 1
make: *** [tools/Unix.mk:509: nuttx] Error 2
make: Target 'all' not remade because of errors.
/github/workspace/sources/nuttx/tools/testbuild.sh: line 257: /github/workspace/sources/nuttx/../nuttx/nuttx.manifest: No such file or directory
  Normalize amber/hello
```

Reply:
And btw, the NuttX kernel also calls exit(), so we do really need to define exit() somehow to the kernel. This is one way to do it. Keep in mind that lib_exit.c is not yet compiled.

Reply:
The AVR build seems to want to pull exit() and _exit() from avr/libgcc unless I make the strong definition for exit() here. 

The nuttx kernel needs / uses exit() also so I think defining it here until the libc version is taken into use is fine.

Reply:
> And btw, the NuttX kernel also calls exit(), so we do really need to define exit() somehow to the kernel. This is one way to do it. Keep in mind that lib_exit.c is not yet compiled.

Let's compile it?